File: src/main/java/com/dansoftware/libraryapp/gui/pluginmanager/PluginManagerWindow.java
package com.dansoftware.libraryapp.gui.pluginmanager;

import com.dansoftware.libraryapp.gui.window.LibraryAppStage;
import javafx.stage.Modality;
import org.jetbrains.annotations.NotNull;

public class PluginManagerWindow extends LibraryAppStage {
    PluginManagerWindow(@NotNull PluginManager pluginManager) {
        super("window.pluginmanager.title", pluginManager);
        initModality(Modality.APPLICATION_MODAL);
        setWidth(750);
        setHeight(550);
    }
}

File: src/main/java/com/dansoftware/libraryapp/gui/pluginmanager/PluginManagerWindow.kt
package com.dansoftware.libraryapp.gui.pluginmanager

import com.dansoftware.libraryapp.gui.window.BaseWindow
import javafx.stage.Modality
import javafx.stage.Window

/**
 * A [javafx.stage.Stage] for showing a [PluginManager]
 *
 * @author Daniel Gyorffy
 */
private class PluginManagerWindow(owner: Window?, view: PluginManager) :
    BaseWindow<PluginManager>("window.pluginmanager.title", view) {
    init {
        initOwner(owner)
        initModality(Modality.APPLICATION_MODAL)
        width = 750.0
        height = 550.0
    }
}